_pages/ontology.md

```diff
@@ -86,11 +86,11 @@ redirect_from:
   - /ontology/type/webpage
 ---
 
-The [CSL specification](https://docs.citationstyles.org/en/1.0.1/specification.html) implies a data model with:
+The [CSL specification](https://docs.citationstyles.org/en/stable/specification.html) implies a data model with:
 
-* [categories](https://docs.citationstyles.org/en/1.0.1/specification.html#appendix-i-categories) to indicate academic fields such as `astronomy` and `biology`,
-* [contributor roles](https://docs.citationstyles.org/en/1.0.1/specification.html#roles) such as `author` and `translator`,
```

Comment:
The `#roles` anchor seems to be gone in the latest spec.

Reply:
Yes, I think that change to be variables is good